litequeue.py

```diff
@@ -482,15 +497,18 @@ def full(self) -> bool:
         else:
             return False
 
-    def prune(self):
+    def prune(self, include_failed: bool = True):
         """
         Delete `done` and `failed` messages. # TODO: maybe not failed ones
```

Comment:
The docstring should be updated to reflect the new arguments.

```suggestion
        Delete `DONE` messages.
        
        If `include_failed` is True, the messages in `FAILED` state will be deleted too.
```